cordova-lib/src/cordova/run.js

```diff
@@ -44,7 +44,7 @@ module.exports = function run(options) {
     }).then(function() {
         return hooksRunner.fire('after_run', options);
     }, function(error) {
-        events.emit('log', 'ERROR running one or more of the platforms: ' + error + '\nYou may not have the required environment or OS to run this project');
+        events.emit('warn', 'ERROR running one or more of the platforms: ' + error + '\nYou may not have the required environment or OS to run this project');
```

Comment:
As in the first comment I'm not sure if we should emit this here because it is not consistent and not very informative. Maybe rethrow with this message. @nikhilkh, what do you think?